models/uniswap/ethereum/uniswap_ethereum_airdrop_claims.sql

```diff
@@ -10,18 +9,18 @@
         post_hook='{{ expose_spells(\'["ethereum"]\',
                                 "project",
                                 "uniswap",
-                                \'["hildobby"]\') }}'
+                                \'["hildobby", "tomfutago"]\') }}'
     )
 }}
 
-{% set uni_token_address = '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984' %}
+{% set uni_token_address = 0x1f9840a85d5af5bf1d1762f925bdaddc4201f984 %}
 
 WITH early_price AS (
     SELECT MIN(minute) AS minute
     , MIN_BY(price, minute) AS price
     FROM {{ source('prices', 'usd') }}
     WHERE blockchain = 'ethereum'
-    AND contract_address= {{uni_token_address}}
+    AND contract_address = from_hex('{{uni_token_address}}')
```

Comment:
`from_hex()` should not be needed on these variables. we had the setup working properly on other spells. it should just be something like:
`{% set uni_token_address = '0x1f9840a85d5af5bf1d1762f925bdaddc4201f984' %}`

then use the below without quotes:
`{{uni_token_address}}`

Reply:
Fixed.